pkg/minishift/provisioner/minishift_provisioner.go

```diff
@@ -111,12 +111,12 @@ func (provisioner *MinishiftProvisioner) Provision(swarmOptions swarm.Options, a
 	}
 	provisioner.EngineOptions.StorageDriver = storageDriver
 
-	fmt.Printf("\n   Setting hostname ... ")
+	log.Info("\n   Setting hostname ... ")
```

Comment:
This is not part of the same issue

Reply:
@gbraad , getting some spinning wheel issue on print it here. See https://github.com/minishift/minishift/issues/1339

Reply:
This is not how it was originally printed. The OK was not on the informative line about setting the hostname.

Reply:
@LalatenduMohanty , WDYT about this change? Right now if we print this, it is altering the expected progress dots behavior. See above comment.

Reply:
I am ok with using `glog.V(2).Infof` or log.Info as we need to fix it anyway with #169 .

Reply:
With `log.Info`, this message will appear in `--show-libmachine-logs` flag enabled as well. Hence, I am in favor of it.